_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,547 @@
+---
+id: ffbc54
+name: No keyboard shortcuts use only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable characters keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][]: 
+ - where the attribute `key` is a [printable character][] key; and
+ - the method `getModifierState` returns `false`; and
+ - is [dispatched][] to an [event target][] within a [HTML document][].
+
+## Expectation
+
+For each test target at least one of the following is true:
+ - (**Disable**:) There is at least one [set of clearly labelled instruments][] to [block the event][blocked event]; or
+ - (**Remap**:) There is at least one [set of clearly labelled instruments][] to [block the event][blocked event] when the [keyboard event][] method `getModifierState` doesn't return `true`; or
```

Comment:
Yes. For example, passed examples 1 and 4 are different because the instrument is different:
- The instrument in passed example 1 always blocks the shortcut after activated. 
- The instrument in passed example 4 only blocks the shortcut if no modifier key is used. 

Testing the expectation of the rule is a process where the operation of the instruments needs to be checked. The outcome of the events is not checked at the time of inspecting the event (the expectation reads "There is at least one set of clearly labelled instruments to block the event" not "There is at least one set of clearly labelled instruments blocking the event")

Reply:
The instrument is different, but any instrument that always blocks the event is also an instrument that blocks when `getModifierKey` is false.

If I test Passed Example 2:
1. I press 'a' without focus, something happens. So **focus** condition is false.
2. I activate the instrument.
3. I press 'a', nothing happens. `getModifierKey` was false for all valid modifier, so **remap** condition is true.
  Is **disable** condition true? I don't know yet. So far, I have no way in differenciating Passed Examples 2 and 4
4. So, I try to press 'Alt+a'. Nothing happen. But I still can't make the difference with Passed Example 4.
5. So I try 'CapsLock+a', … (14 modifiers keys total (and I'm not sure whether there can be several at the same time to a whopping 2¹⁴ combinations)). After trying all of these, I know that the event is always disabled, not just for some modifiers keys.

Now, why would I do steps 4-5. After step 3, I already now that the rule pass. So, as a tester I will not make any difference between Passed Example 2 and 4 and I will never know if the **disable** condition is true or not.

I suggest we ditch the **disable** condition, keep Passed Example 2 and mention in the description that "disabled" implies "disabled when `getModifierKey` is false".

Reply:
We do leave out a scenario where the implementation disables the event and the tester checks the instrument by pressing the same key but now with a modifier key. There will be no outcome because the event is disabled. The expectation says nothing about it. Of course, if the tester tests the same key without the modifier there will be no outcome, which is according to the expectation and the rule passes.
I guess your suggestion of adding an explanation to one of the examples could do the trick. I'll take the disable condition out of the expectation.